ngraph/frontend/tensorflow/include/tensorflow_frontend/model.hpp

```diff
@@ -19,17 +20,12 @@ class TF_API InputModelTF : public InputModel {
     class InputModelTFImpl;
     std::shared_ptr<InputModelTFImpl> _impl;
 
-public:
-    // TODO: move to private once GraphTranslation will be a part of FrontEndTF component
     std::vector<std::shared_ptr<OpPlaceTF>> get_op_places() const;
```

Comment:
now we can get this information from GraphTranslation, right?

Reply:
yep